apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -436,20 +447,18 @@ type FlowCollectorFLP struct {
 	// +kubebuilder:default:=Flows
 	LogTypes *FLPLogTypes `json:"logTypes,omitempty"`
 
+	// List of additional features to enable. Possible values are:<br>
+	// - `MultiCluster`: allows multi-cluster awareness by labelling flows in this cluster with the cluster name.<br>
+	// - `Zone`: allows availability zone awareness by labelling flows with their source and destination zones.
+	// This feature requires the "topology.kubernetes.io/zone" label to be set on nodes.<br>
+	// +optional
+	Features []ProcessorFeature `json:"features,omitempty"`
```

Comment:
The approach of having a list of feature is fine as long as we don't want some feature enabled by default. Then it can becomes a bit misleading for the user.

Lets say we want Zone enabled by default. If a user wants to enable Multicluster, Feature will have to go from not set to ["Zone", "MultiCluster"]. Many users could inadvertently disable the default feature.